jenkins/Jenkinsfile_noms

```diff
@@ -0,0 +1,85 @@
+pipeline {
+  agent {
+        node {
+            label 'team:makerspace'
+        }
+    }
+    environment {
+     GA_AUTH_FILE = credentials('GOOGLE_ANALYTICS_SERVICE_JSON')   
+     GA_CLIENTID_FILE = credentials('GOOGLE_ANALYTICS_CLIENTID_JSON')
+    }
+    parameters {
+        gitParameter name: 'BRANCH_TAG',
+                     type: 'PT_BRANCH_TAG',
+                     defaultValue: 'v0.0.6'
+        choice(choices: ['test', 'prod'], description: 'Tier to deploy tiles to', name: 'TIER')
+    }
+    triggers {
+        cron('H(45-59) 7 * * *')
```

Comment:
We do have the parameterizedCron plugin now, so don't have to rely on defaults for the scheduled setting: https://code.usgs.gov/wma/wp/forecasted-met-drivers/-/blob/master/jenkins/Jenkinsfile#L16

Reply:
forgot about this but will look at it momentarily.